### PR TITLE
CAL-293 Move itests to integration-test phase and run in single container.

### PR DESCRIPTION
**Pull request by clockard**
#### What does this PR do?
Moves the itests to the integration-test phase and also updates the itests so they run in a single container. Running in a single container made the itest runtime go from ~20 min to ~ 6 min (building locally)
Also fix an itest in ImagingTest that was causing some flakiness.
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@brendan-hofmann @mcalcote 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).

@figliold
@shaundmorris

#### How should this be tested?
Verify itest pass
#### What are the relevant tickets?
[CAL-293](https://codice.atlassian.net/browse/CAL-293)
#### Checklist:
- [ ] Documentation Updated
	- [ ] Change Log Updated
- [ ] Update / Add Unit Tests
- [x] Update / Add Integration Tests

### CODE REVIEW

File: distribution/test/itests/test-itests-alliance/src/test/java/org/codice/alliance/test/itests/AllianceAppsTest.java [hunk 1]
```diff
@@ -1,99 +0,0 @@
-/**
```
Comment (clockard) [inline distribution/test/itests/test-itests-alliance/src/test/java/org/codice/alliance/test/itests/AllianceAppsTest.java]: By adding the apps to the default list to start for the itests we already perform this test. No need to run it again.

**Comment by clockard** [comment]
build now

**Comment by clockard** [comment]
build now

**Comment by clockard** [comment]
build now

**Comment by clockard** [comment]
build now

File: distribution/test/itests/test-itests-common/src/main/java/org/codice/alliance/test/itests/common/AbstractAllianceIntegrationTest.java [hunk 2]
```diff
@@ -32,7 +33,26 @@
 public abstract class AbstractAllianceIntegrationTest extends AbstractIntegrationTest {
 
     public static final String[] DEFAULT_ALLIANCE_APPS =
-            {"catalog-app", "solr-app", "spatial-app", "security-app"};
+            {"catalog-app", "solr-app", "spatial-app", "security-app", "imaging-app", "video-app", "nsili-app"};
```
Comment (shaundmorris) [inline distribution/test/itests/test-itests-common/src/main/java/org/codice/alliance/test/itests/common/AbstractAllianceIntegrationTest.java]: Do we need a comment so others no know how to maintain this list?

Comment (clockard) [inline distribution/test/itests/test-itests-common/src/main/java/org/codice/alliance/test/itests/common/AbstractAllianceIntegrationTest.java]: Added comment

**Comment by clockard** [comment]
build now

**Comment by clockard** [comment]
build now

**Comment by clockard** [comment]
build now

**Pull request merged**